guardowl/protocols.py

```diff
@@ -571,7 +445,7 @@ def run_hipen_protocol(
     :param implementation: The implementation to use.
```

Comment:
Mixture of Google and numpy style docstrings in classes and functions. Some variables missing in this docstring.

Reply:
thanks! that has been resolved!